lms/templates/courseware/progress.html

```diff
@@ -168,21 +169,26 @@ <h3 class="hd hd-3" id="chapter_${loop.index}">${ chapter['display_name']}</h3>
                                 %for section in chapter['sections']:
                                     <div>
                                         <%
-                                        earned = section.all_total.earned
-                                        total = section.all_total.possible
-
-                                        percentageString = "{0:.0%}".format(section.percent_graded) if earned > 0 and total > 0 else ""
+                                        if section.override:
```

Comment:
I think if you rebase onto master, you'll be able to get rid of this `if` block - you basically need to do this right now because of a bug.  When that bug is eliminated, the subsection_grade object's `graded_total` and `all_total` attributes should take any existing overrides into account without you having to worry about it here.

Reply:
I took a look into removing this.  `all_total` doesn't appear to include the override value however if I switch to using the `graded_total` it does.  I'll make the switch to that value since it gets rid of both this conditional and the need for the `graded` variable I had set